backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/renderInvoiceAddress.feature

```diff
@@ -19,14 +19,14 @@ Feature: Render invoice address
       | pl_so      | ps_1                          | DE                        | EUR                 | price_list_so | true  | false         | 2              |
     And metasfresh contains M_PriceList_Versions
       | Identifier | M_PriceList_ID.Identifier | Name   | ValidFrom  |
-      | plv_so     | pl_so                     | plv_so | 2022-02-01 |
+      | plv_so     | pl_so                     | plv_so | 2022-01-20 |
     And metasfresh contains M_ProductPrices
       | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_TaxCategory_ID.InternalName |
-      | pp_product | plv_so                            | addr_test_product       | 10.0     | Normal                        |
+      | pp_product | plv_so                            | test_product_26_02      | 10.0     | Normal                        |
 
     And metasfresh contains C_BPartners:
```

Comment:
```suggestion
    # create bpartner with invoice-rule "immediate", because we need jsut an invoice without a shipment
    And metasfresh contains C_BPartners:
```